components/arm/arm_test.go

```diff
@@ -6,8 +6,8 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/go-viper/mapstructure/v2"
 	"github.com/golang/geo/r3"
-	"github.com/mitchellh/mapstructure"
```

Comment:
This library has been archived and the one that replaces it has been given the blessing of the author as the new one to use.  In investigating this ticket I found that the library had shortcomings in inferring certain types to decode (namely maps).  There has been discussion of making this type of improvement on the new library so I think we should make this transition now